protocol/x/clob/keeper/equity_tier_limit.go

```diff
@@ -149,8 +145,9 @@ func (k Keeper) ValidateSubaccountEquityTierLimitForNewOrder(ctx sdk.Context, or
 	if lib.MustConvertIntegerToUint32(equityTierCount) >= equityTierLimit.Limit {
 		return errorsmod.Wrapf(
 			types.ErrOrderWouldExceedMaxOpenOrdersEquityTierLimit,
-			"Opening order would exceed equity tier limit of %d. Order id: %+v",
+			"Opening order would exceed equity tier limit of %d. Order count: %d, order id: %+v",
```

Comment:
Wdyt of adding the TNC in this error?

Reply:
Getting TNC is actually quite a bit of work in terms of CPU time. if we only expect a small amount of orders to fail equity tier limit, then I am fine with adding. but if we expect a large number of subaccounts failing, imo we should just leave it out

Reply:
probably better to add the entire subaccount instead of TNC since that's way cheaper (we can calculate TNC offline if there are issues)

Reply:
We already calculate net collateral [here](https://github.com/dydxprotocol/v4-chain/blob/79b327ce906ffd74219927be62b6079eaa2bdfd5/protocol/x/clob/keeper/equity_tier_limit.go#L88) in this function though right? So no extra computation is required.